src/main/java/com/rakovets/course/java/core/practice/operators/Task02.java

```diff
@@ -30,8 +30,7 @@ public static void main(String[] args) {
      * @param carryingCapacity грузоподъемность грузовиков
      */
     static int getAmountRemainingCargo(int cargo, int carryingCapacity) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+
+        return cargo%carryingCapacity;
```

Comment:
Violation of Code Convention
```suggestion
        return cargo % carryingCapacity;
```